lib/screens/app/wallet/receive.dart

```diff
@@ -539,6 +541,18 @@ class _ReceiveFormState extends State<ReceiveForm> {
                 }
               },
             ),
+            Align(
+                alignment: Alignment.topLeft,
+                child: Padding(
+                    padding: EdgeInsets.fromLTRB(17, 5, 0, 0),
```

Comment:
Why 17? 
Its seems odd to use an odd number :) 
16 should be better or 18
Normally we use increments of 8

8-16-24-32
For paddings